engine/src/main/java/org/terasology/logic/debug/MovementDebugCommands.java

```diff
@@ -233,22 +236,52 @@ public String stepHeight(@Sender EntityRef client, @CommandParam("height") float
         return "";
     }
 
+    private float getJumpSpeed(float amount){
+        double d = (double)amount;
+        return Math.round(pow(d,0.55) * 8.45f);
+    }
+    
+    private float getInteractionRange(float amount){
+        double d = (double)amount;
+        return Math.round(pow(d,0.6) * 3.9f);
+    }
+
+    private float getRunFactor(float amount){
+       return Math.round(5.5f + (6.4f * amount) - (0.06f * amount * amount)) / 10f;
+    }
+
     @Command(shortDescription = "Sets the height of the player", runOnServer = true,
             requiredPermission = PermissionManager.CHEAT_PERMISSION)
     public String playerHeight(@Sender EntityRef client, @CommandParam("height") float amount) {
         try {
             ClientComponent clientComp = client.getComponent(ClientComponent.class);
             CharacterMovementComponent move = clientComp.character.getComponent(CharacterMovementComponent.class);
-            if (move != null) {
+            CharacterComponent charComp = clientComp.character.getComponent(CharacterComponent.class);
+            EntityRef player = clientComp.character;
+            GazeMountPointComponent gazeMountPointComponent = player.getComponent(GazeMountPointComponent.class);
+
+            float defaultHeight = 1.6f;
```

Comment:
These should not be re-defined here. When the default values in CharacterMovementComponent are changed these will be outdated, and will likely cause some weird behavior and bugs.

Instead, `CharacterMovementComponent` should expose 

```java
    public  static final float DEFAULT_HEIGHT = 1.6f;
    public  static final float DEFAULT_JUMP_SPEED = 10.f;
```

Reply:
i just looked at the `restoreSpeed` command. This is how it accesses default values `CharacterMovementComponent moveDefault = prefab.get().getComponent(CharacterMovementComponent.class);` for instance ` move.jumpSpeed = moveDefault.jumpSpeed;` Should i do this?